authguard.service

```diff
@@ -7,6 +7,16 @@ Wants=network-online.target
 Type=simple
 ExecStart=/opt/authguard -web.listen-address=:8080 -web.proxy-to=127.0.0.1:9090 -user=username -pass=password -crt=/etc/ssl/private/tmpcert.pem -key=/etc/ssl/private/tmpkey.pem
 Restart=always
+
+# optional systemd hardening-options Documentation about these can be found under
+# man 5 systemd.exec
```

Comment:
ahm, you removed exactly that linebreak I'd have kept and *not* that one I wanted to get rid of^^

Can you do it the other way round, please?